.github/CONTRIBUTING.md

```diff
@@ -65,11 +68,186 @@ Pull Requests complies with these rules:
     file][Markdown linter config]
   - additionally, links should be in the reference format, not in the in-text format
 
+## Git mini-tutorial
+
+Before starting the tutorial, it is important to clarify definitions of the
+following for beginners:
+
+- git commandline tool
+  - for running "git commands" with
+- Git SCM (source code management) format
+  - Git repositories are directories of this format
+- Github
+  - projects are hosted at this publicly accessible collaboration platform which
+    is built around Git repositories
+
+This is a "just the steps" mini-tutorial (text with preceding `##` is either for
+you to replace, or instructions to follow). It is aimed at first-time users of
+Git, or relatively new users wanting to know the steps this project uses in the
+context of Github. If any step is confusing it should be easy to find details &
+explanations by searching Github documentation, or googling the command and
+especially looking at stackoverflow answers that come up for it. The
+instructions below are for the example-case of a first-time contributor adding a
+language to the translation-table:
+
+1. Ensure you have the [git][Git] commandline tool installed and usable in a
+   shell. If on Windows and you haven't already installed a "POSIX compatible"
+   shell ("command line"), the simplest thing is to install
+   [the bundled git package][Git windows bundle] which includes its own shell.
+1. In your browser, logged into Github, in the top-right corner of
+   [this repo][Project main page], click the `Fork` icon which will create your
+   own fork.
+1. In the shell do the following. Setting the variables at the beginning is just
+   to make this tutorial more readable, you can obviously enter the text
+   directly each time instead of using variables if you prefer:
+    ```sh
+    common_dir=## Some common directory to hold your repositories.
+    user=## Your github username.
+    lang=## Your translation language.
+
+    mkdir -p "${common_dir}"
+    cd "${common_dir}"

```

Comment:
I can't say for sure, but I think I would be a little bit confused by this as a newbie: I like the idea of using shell variables to avoid repeating text, but I would make it more clear that in line 104-106 you should actually replace the comment with a value (mybe add "Remove the ## and replace this comment with...", while lines like 108 should be entered as they are because the shell will insert the right value at the right place for you.